framework/src/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -851,6 +851,16 @@ public Boolean visitUnion_Union(
         return visitUnionSubtype(subtype, supertype, visited);
     }
 
+    @Override
+    public Boolean visitUnion_Wildcard(
+            AnnotatedUnionType subtype, AnnotatedWildcardType supertype, VisitHistory visited) {
+        // } catch (RuntimeException | IOException e) {
```

Comment:
A comment that this is an example, like you have in the other methods, would make this clearer. I first thought this is some leftover code.